### DISCUSSION TITLE
Desktop UI resizing/horizontal scrollbar/open email in a new window

### DISCUSSION THREAD

**Discussion by pgeyfman** [tutao/tutanota#6217] [Ideas]
As the title says, there needs to be a way of making more space in the email preview window when viewed on smaller resolution laptop screens.  As of now, the emails that are formatted wide do not fit into the preview window and there is no way of displaying them fully in the desktop client on Linux.  The workaround is to open the email in the browser and use the zoom function to make everything smaller.

**Comment by pgeyfman** [comment]
Alternatively, if an IMAP service can be added so that a different email client can be used, that would be great too.

**Comment by charlag** [comment]
The desktop app can be scaled like the browser but yes changing the column size would be great

**Reply by pgeyfman** [reply]
How do you scale the desktop client? I couldn't figure it out. Using the Linux client.

**Reply by ganthern** [reply]
you can do it with holding `ctrl` + using the mouse wheel / touch pad equivalent. reset to 100% is with `ctrl` + `0`

**Reply by pgeyfman** [reply]
Thank you!! That solves my issue!! Although, some flexibility in customizing the app view would still be welcome.
